src/app/[locale]/seller/sale-items/[id]/page.tsx

```diff
@@ -43,14 +47,32 @@ export default function Page({ params }: { params: { id: string } }) {
     const getSellerData = async () => {
       try {
         const data = await fetchSingleSeller(sellerId); //'testme'
-        setSeller(data);  // Ensure this is a single object, not an array
+        setSeller(data.business_info);  // Ensure this is a single object, not an array
```

Comment:
It seems a bit redundant to have these null conditionals since `useState` already initializes the state variables to null.